github_discovery.py

```diff
@@ -314,6 +314,40 @@ def get_slack_channel_name_by_id(slack_channel_id):
   log.debug(f'Slack channel name for {slack_channel_id} is {slack_channel_name}')      
   return slack_channel_name
 
+def find_channel_by_severity_label(alert_severity_label):
+    try:
+        response = requests.get(ALERTMANAGER_URL, verify=False)
```

Comment:
I think we should move this to the outer loop and save the results in a variable.  That way it wont be calling the alertmanager endpoint for every component processed.